compiler/src/iree/compiler/Codegen/TransformDialectStrategies/CPU/Common.cpp

```diff
@@ -76,6 +76,22 @@ std::pair<Value, Value> mlir::iree_compiler::cpu::buildCommonTrailingStrategy(
 // user-friendliness.
 //===----------------------------------------------------------------------===//
 
+/// Placeholder to encode fixed reductions that should take finer-grained
+/// precedence over other heuristics. In the future, this could be lifted to
+/// e.g. `cpuModel` or higher up in some transform dialect database summary of
+/// "known good things".
+static FailureOr<ReductionConfig> applyKnownGoodReductionConfigurations(
+    const transform_ext::MatchedReductionCaptures &captures,
+    const CPUModel &cpuModel) {
+  int64_t reductionSize = captures.reductionOpSizes.back();
+  if (cpuModel.model == CPUModel::kXeonGold6154) {
+    if (captures.reductionOutputElementalTypeBitWidth == 32) {
+      if (reductionSize) return ReductionConfig{32};
```

Comment:
Nit: this config looks too much like magic numbers. Can we at least have `/*foo=*/` comments in the constructor?